packages/service-core/src/sync/sync.ts

```diff
@@ -187,7 +207,7 @@ async function* streamResponseInner(
         // 3. However, the new checkpoint does not contain any new data for this connection, so nothing further is synced.
```

Comment:
I might be missing something here, but is this comment still valid? It seems like we are no longer disabling interrupting checkpoints.

Reply:
Good catch, the comment should be removed.